internal/caller/wgconfig.go

```diff
@@ -0,0 +1,192 @@
+package caller
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+
+	"howett.net/plist"
+)
+
+var (
+	tempFile       = "./config_temp.mobileConfig"
+	filename       = "./bh-wireguard.mobileConfig"
+	identifier     = "com.betterhelp.wireguard.locksmith"
+	badEncoding    = "&#x[1-9A-Z];"
+	tunnelTemplate = `[Interface]
+PrivateKey = %s
+Address = 10.10.1.%s/24
+	
+[Peer]
+PublicKey = %s
+Endpoint = demo.wireguard.com:12912
```

Comment:
Should this and the next lline be real endpoints, or at least template-able?

Reply:
It should definitely be templated information. I imagine in the final product the Client will pass this and some remaining information down to the config it needs generated. I was using a demo template from wg for feasibility testing with installation, but I can make all configured information a format option now.